app/views/layouts/_right_section.html.haml

```diff
@@ -0,0 +1 @@
+= react('RightSection', { :customLogo => ::Settings.server.custom_logo, :currentUser => current_user, :helpMenu => menu_to_json(:help),:opsExplorerAllowed => ApplicationHelper.role_allows?(:feature => 'ops_explorer', :any => true), :applianceName => appliance_name, :miqGroups => current_user.miq_groups, :currentGroup => current_group, :userMenu => menu_to_json(:top_right) }, {:class => "nav navbar-nav navbar-right navbar-iconic"}, 'ul')
```

Comment:
split this into 1 parameter per line please :)

And you'll need a map over `miqGroups` to `to_s` the ids